R/ResultFlagsDependent.R

```diff
@@ -244,17 +244,17 @@ InvalidSpeciation <- function(.data, clean = c("invalid_only", "nonstandardized_
   #                   .after = "MethodSpecificationName"
   #   )
   
-  # if all rows are "Valid", return input unchanged
+  # if all rows are "Valid", return input with flag column
   if (any(c("Nonstandardized", "Invalid") %in%
           unique(check.data$TADA.MethodSpeciation.Flag)) == FALSE) {
-    print("All data is valid, therefore the function cannot be applied.")
-    .data = OrderTADACols(.data)
+    print("All characteristic/method speciation combinations are valid in your dataframe. Returning input dataframe with TADA.MethodSpeciation.Flag column.")
```

Comment:
Suggestion: Returning input dataframe with added TADA.MethodSpeciation.Flag column for tracking.

Reply:
I will double check but I'm pretty sure check.data in these functions is simply the input dataframe with the flag column added. Do you want me to rename check.data to just .data?